### PR TITLE
Add Flame

### PR DESCRIPTION
**Pull request by impulse**
Old PR was closed due to master -> main branch rename. 

Reference for existing discussion: https://github.com/sindresorhus/awesome/pull/1785

### CODE REVIEW

**Comment by sindresorhus** [comment]
> A curated list of Games, Libraries, and Articles related to the Flame Engine for Flutter

Many words here that should not be capitalized.

---

The license file is not valid. It needs to contain the whole license.

---

> Do not add the license name or text to the readme. GitHub already shows the license name at the top of the repo.

---

Linkify `Flame` in `Flame is a minimalist 2D game engine for Flutter that provides` to its website.

---

Non-english content should be in a separate Markdown file.

File: readme.md [hunk 1]
```diff
@@ -391,6 +391,7 @@
 - [libGDX](https://github.com/rafaskb/awesome-libgdx#readme) - Java game framework.
 - [PlayCanvas](https://github.com/playcanvas/awesome-playcanvas#readme) - Game engine.
 - [Game Remakes](https://github.com/radek-sprta/awesome-game-remakes#readme) - Actively maintained open-source game remakes.
+- [Flame](https://github.com/flame-engine/awesome-flame#readme) - Game engine.
```
Comment (sindresorhus) [inline readme.md:394]:
```suggestion
- [Flame](https://github.com/flame-engine/awesome-flame#readme) - Game engine for Flutter.
```

**Comment by impulse** [comment]
@sindresorhus I have applied all of your suggested changes in https://github.com/flame-engine/awesome-flame/pull/36.

Renaming master to main will be done ASAP. I don't have the repo rights to do that right now. 
Chinese translations will be added in a separate file later. Just waiting for a native speaker to look over my changes.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1276832322390614016

**Pull request merged**